apps/src/templates/instructions/InlineHint.jsx

```diff
@@ -27,14 +28,31 @@ const InlineHint = React.createClass({
     }
   },
 
+  onVideoClick() {
+    firehoseClient.putRecord(
+      'analysis-events',
+      {
+        study: 'hint-videos',
+        event: 'click',
+        data_string: this.props.video.key,
+      },
+      { includeUserId: true }
+    );
+  },
+
   render() {
     return (
       <ChatBubble borderColor={this.props.borderColor} ttsUrl={this.props.ttsUrl} ttsMessage={this.props.ttsMessage}>
```

Comment:
Do we also want to track that the user asked to see the hint?  I think this data is already in the DB, but maybe easier to correlate with hint video clicks if we log it to Firehose as well.  @bencodeorg any preference?